app/bundles/LeadBundle/Form/Type/CompanyMergeType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @copyright   2015 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\LeadBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+/**
+ * Class CompanyMergeType.
+ */
+class CompanyMergeType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array                $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add(
+            'company_to_merge',
+            'company_list',
+            [
+                'multiple'    => false,
```

Comment:
The label '`Companies` which is default doesn't really apply here. Either add a custom label or add `'label' => false,` to hide it (the Modal header I think suffices enough for the input to not have a label).